docs/how-to-guides/column-mapping.md

```diff
@@ -0,0 +1,66 @@
+# Column mapping
+
+In this document, we will introduce how to map inputs with column mapping when running a flow.
+
+## Column mapping introduction
+
+Column mapping is a mapping from flow input name to specified values.
+If specified, the flow will be executed with provided value for specified inputs.
+The following types of values in column mapping are supported:
+
+- `${data.<column_name>}` to reference from your test dataset.
+- `${run.outputs.<output_name>}` to reference from your flow output.
```

Comment:
Here the wording `your flow output` seems to indicate the output of **this flow**, rather than the "referenced" flow as stated [here](https://microsoft.github.io/promptflow/reference/pf-command-reference.html#id16).

We probably need to call out that the prerequisite of this reference is `--run` exists in `pf run`. And the scenario is to receive outputs from another flow.

Reply:
Good catch! Updated.